app/helpers/point_history_helper.rb

```diff
@@ -1,9 +1,25 @@
 module PointHistoryHelper
-  def detailed_point_history(pointrequests, semester)
+  def user_detailed_point_history(pointrequests, semester)
     semester_string = semester.to_s
     pointrequests.select { |pr| pr.accepted? && pr.evaluation.semester == semester_string }
                  .each do |pointrequest|
       yield DetailedPointHistory.new(pointrequest)
     end
   end
+
+  def evaluations_to_semester_options(evaluations)
+    evaluations.collect do |evaluation|
+      [evaluation.date_as_semester.to_readable, evaluation.semester]
+    end
+  end
+
+  def group_detailed_point_history(evaluation)
+    evaluation.point_requests
+              .includes(:user)
+              .sort { |a, b| a.user.transliterated_full_name <=> b.user.transliterated_full_name }
```

Comment:
I think this solution is not give the correct sequence all time.
ex:
`a = {1 => "kolok", 2 => "ütő", 3 => "uto", 4 => "kölök"}`
 if they are encoded in UTF-8, then the
`a.sort{|a1, a2| a1[1] <=> a2[1]}`
will return
`=> [[4, "kolok"], [1, "kölök"], [3, "uto"], [2, "ütő"]]`
but the
`a.sort{|a1, a2| I18n.transliterate(a1[1]) <=> I18n.transliterate(a2[1])}`
will return 
`=> [[1, "kolok"], [4, "kölök"], [2, "ütő"], [3, "uto"]]`

So this depends on the original order.